tests/hmm/hmumu_utils.py

```diff
@@ -2874,16 +2899,27 @@ def multiply_all(weight_list):
         ret *= w
     return ret
 
-def compute_lepton_sf(leading_muon, subleading_muon, lepsf_iso, lepsf_id, lepsf_trig, use_cuda, dataset_era, NUMPY_LIB, debug):
+def multiply_all_trig(weight_list):
```

Comment:
maybe we can put a comment here that this computes (as I understand) `1 - (1 - w1)(1 - w2)(1 - w3)...`